Marlin/src/feature/host_actions.cpp

```diff
@@ -182,6 +182,7 @@ void HostUI::action(FSTR_P const fstr, const bool eol) {
           case 1: // "Continue" / "Disable Runout" button
             #if BOTH(M600_PURGE_MORE_RESUMABLE, ADVANCED_PAUSE_FEATURE)
               pause_menu_response = PAUSE_RESPONSE_RESUME_PRINT;  // Simulate menu selection
+              hostui.prompt_end();
```

Comment:
When a prompt sends a response we don't need to tell it to go away. It should already have closed itself.